pkg/kubelet/kubelet.go

```diff
@@ -2952,6 +2953,27 @@ func disallowResizeForSwappableContainers(runtime kubecontainer.Runtime, desired
 	return false, ""
 }
 
+// Get pod resize info in json format
+func getPodResizeInfo(pod *v1.Pod) (string, error) {
```

Comment:
I'd suggest to move this function to pkg/kubelet/allocation/allocation_manager.go